changelog/0043-optionally-configure-auditcare-db.yml

```diff
@@ -0,0 +1,42 @@
+title: Optionally configure auditcare db
+key: optionally_configure_auditcare_db
+date: 2021-03-23
+optional_per_env: yes
+min_commcare_version:
+max_commcare_version:
+context: |
+  Some enterprise deployments of CommCare use a backend feature called "auditcare" as part of
+  their audit logging strategy. Historically it has used CouchDB as its data backend,
+  but it is being switched to use PostgreSQL instead. If you care about this feature then
```

Comment:
What do you think of adding another sentence in here? Something like

_Auditcare is enabled by default, so it is active unless you went out of your way to disable it when you configured CommCare._

Reply:
Like this?

```suggestion
  but it is being switched to use PostgreSQL instead.
  Auditcare is enabled and minimally configured by default,
  so it is active unless you went out of your way to disable it when you configured CommCare.
  If you care about this feature then
```